### DISCUSSION TITLE
Large Amount of Files in Folder

### DISCUSSION THREAD

**Discussion by cbeno68** [prasathmani/tinyfilemanager#914] [Q&A]
Hello there, 

I want to start off by saying this is a great project and I am beyond happy that I found something so powerful yet so simple to use. I wanted to ask if there was an issue with folders loading if they have a huge amount of files? I have a hand full of folders with close to a thousand files in it. The files are small but there is just a ton of them. Thank you!

Chris

**Comment by prasathmani** [comment]
no issue, it will work perfect but little slower.

**Reply by cbeno68** [reply]
Anyway to cut that down? I have also noticed the page stops creating the files and links at a certain point and doesn't finish the footer that shows the amount of files and data on the page.
![image](https://user-images.githubusercontent.com/20207080/208454175-10ba8abd-9aa2-42db-bf13-3365a63807a1.png)
